packages/formatting/README.md

```
@@ -0,0 +1,300 @@
+  verbose: true,
+});
+
+if (result.success) {
+  console.log('Setup completed!');
+  console.log('Generated configs:', result.data.configs);
+}
+```
+
+## Presets
+
+### Standard (Default)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)